### PR TITLE
Add support for random_shuffle argument in test_RN50_data_pipeline

### PR DESCRIPTION
**Pull request by joehandzik**
Implementation from Craig Tierney of NVIDIA

Signed-Off-By: Joe Handzik <jhandzik@nvidia.com>

#### Why we need this PR?
- New feature used in testing that would be helpful to have upstream

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     *[ Added random_shuffle argument support via command line argument ]*
 - Affected modules and functionalities:
     *[ Just the python test module for the RN50 pipeline ]*
 - Key points relevant for the review:
     *[ Minor change, not a lot of work unless you object to the change ]*
 - Validation and testing:
     *[ Used internally for testing ]*

**JIRA TASK**: *[NA]*

### CODE REVIEW

File: dali/test/python/test_RN50_data_pipeline.py [hunk 1]
```diff
@@ -201,6 +206,8 @@ def define_graph(self):
                     help='Cache type')
 parser.add_argument('--reader_queue_depth', default=1, type=int, metavar='N',
                     help='prefetch queue depth (default: 1)')
+parser.add_argument('--read_shuffle', default=0, type=int, metavar='N',
```
Comment (JanuszL) [inline dali/test/python/test_RN50_data_pipeline.py:209]:
```suggestion
parser.add_argument('--read_shuffle', action='store_true',
```

File: dali/test/python/test_RN50_data_pipeline.py [hunk 2]
```diff
@@ -201,6 +206,8 @@ def define_graph(self):
...
+                    help='Shuffle data when reading (default: 0)')
```
Comment (JanuszL) [inline dali/test/python/test_RN50_data_pipeline.py:210]:
```suggestion
                    help='Shuffle data when reading')
```

File: dali/test/python/test_RN50_data_pipeline.py [hunk 3]
```diff
@@ -228,8 +235,10 @@ def define_graph(self):
 STICK_TO_SHARD = True if CACHED_DECODING else False
 SKIP_CACHED_IMAGES = True if CACHED_DECODING else False
 
-print("GPUs: {}, batch: {}, workers: {}, prefetch depth: {}, loging interval: {}, fp16: {}, NHWC: {}"
-      .format(N, BATCH_SIZE, WORKERS, PREFETCH, LOG_INTERVAL, FP16, NHWC))
+READ_SHUFFLE=bool(args.read_shuffle)
```
Comment (JanuszL) [inline dali/test/python/test_RN50_data_pipeline.py:238]:
```suggestion
READ_SHUFFLE=args.read_shuffle
```

File: dali/test/python/test_RN50_data_pipeline.py [hunk 4]
```diff
@@ -228,8 +235,10 @@ def define_graph(self):
...
+
+print("GPUs: {}, batch: {}, workers: {}, prefetch depth: {}, loging interval: {}, fp16: {}, NHWC: {}, READ_SHUFFLE: {}"
+      .format(N, BATCH_SIZE, WORKERS, PREFETCH, LOG_INTERVAL, FP16, NHWC,READ_SHUFFLE))
```
Comment (JanuszL) [inline dali/test/python/test_RN50_data_pipeline.py:241]:
```suggestion
      .format(N, BATCH_SIZE, WORKERS, PREFETCH, LOG_INTERVAL, FP16, NHWC, READ_SHUFFLE))
```

**Comment by joehandzik** [comment]
Good suggestions, updates made. Thanks!

**Comment by JanuszL** [comment]
!build

**Pull request merged**